shared/middleware/net_sim_api.rb

```diff
@@ -375,6 +380,18 @@ def redis_url
     CDO.geocoder_redis_url || 'redis://localhost:6379'
   end
 
+  # Returns an array of URLs for the redis read replicas in the current
+  # configuration.  Returns an empty array if no read replicas are
+  # available.
+  #
+  # @return [Array]
+  def redis_read_replica_urls
+    urls = []
+    urls.push(CDO.geocoder_read_replica_1) unless CDO.geocoder_read_replica_1.nil?
+    urls.push(CDO.geocoder_read_replica_2) unless CDO.geocoder_read_replica_2.nil?
+    urls
+  end
+
```

Comment:
[A replication group can have a maximum of 5 read replicas](http://docs.aws.amazon.com/AmazonElastiCache/latest/UserGuide/Replication.AddReadReplica.html). If it's not too much trouble, I think it would be helpful to have this code support the maximum number from the start, so it would be easier to scale up by adding new nodes later, and only changing configuration (rather than deploying new application code).